packages/components/app/styles/components/form/radio-card.scss

```diff
@@ -0,0 +1,131 @@
+// FORM > RADIO-CARD
+
+// RadioCard Group
+
+.hds-form-group__radio-cards {
+  .hds-form-group__control-fields-wrapper {
+    gap: 16px;
```

Comment:
we can't use `gap` yet because of the browsers versions that we need to support (see thread here: https://hashicorp.slack.com/archives/C11JCBJTW/p1635973292134000)

_(I've asked in Slack again, but I don't expect much difference from one year ago)_

Reply:
Just wanted to comment... From the Slack conversations it looks like John Cowen would be pretty open to updating the browsers they support so we could use flex gap.

https://hashicorp.slack.com/archives/C11JCBJTW/p1636022796147300?thread_ts=1635973292.134000&cid=C11JCBJTW
_"... Consul agreed to use a ~2 years back support, and thats what I'll be PRing today (hopefully). **But that is not set in stone at all.** It sounds like you have a straightforwards workaround, but **if it becomes too much of a workaround please reach out**"_

Reply:
I forgot about this open discussion. Until we clarify whether or not flex `gap` is something we're confident using across the products I switched to a `margin`-based approach.